packages/framer-motion/src/gestures/use-tap-gesture.ts

```diff
@@ -59,6 +59,9 @@ export function useTapGesture({
         if (isPressing.current) return
         isPressing.current = true
 
+        /**
+         * TODO: Make passive if tap handlers dont exist?
```

Comment:
Also here, why aren't we doing it already?

Reply:
And here I forgot to implement too